src/fides/api/ctl/routes/datamap.py

```diff
@@ -100,6 +105,7 @@
 async def export_datamap(
     organization_fides_key: str,
     response: Response,
+    include_deprecated_columns: bool = False,
```

Comment:
I think this helps a lot, I noticed in earlier testing that the open-source export was broken with the added field and this resolves that 🙌🏽